solr/solrj/src/java/org/apache/solr/common/util/IOUtils.java

```diff
@@ -16,15 +16,14 @@
  */
 package org.apache.solr.common.util;
 
-import java.io.Closeable;
 import java.lang.invoke.MethodHandles;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class IOUtils {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  public static void closeQuietly(Closeable closeable) {
+  public static void closeQuietly(AutoCloseable closeable) {
```

Comment:
fwiw it was not fully backward compatible https://github.com/SearchScale/dataimporthandler/issues/91 sigh

Reply:
Ugh.  Fixing: https://github.com/apache/solr/pull/3199